src/commands/configure/domain.rb

```diff
@@ -13,7 +13,7 @@ def setup(_args, _options); end
         protected
 
         def answers_file
-          File.join(config.answer_files_path, 'domain.yaml')
+          config.domain_answers_file
```

Comment:
Not sure what you mean - it's domain-specific because it returns the domain answers file and not any other. I think there's an advantage to having a method which takes no arguments and returns the correct file, rather than a method we need to ensure we pass the correct relative path to to get the domain answers file. Of course any of these answer file methods can use the same method to get the full path behind the scenes.